content/notifications/reference/architecture/notifications/_index.en.md

```diff
@@ -87,13 +87,16 @@ is implemented and used by all services that publish to Kafka.
 
 **Altinn APIs:**
 
-The Notification microservice implements multiple API clients for the Altinn API.
-The clients are used to retrieve recipient data.
+The Notification microservice implements multiple API clients for communication with other services.
+The clients are used to retrieve recipient data and to authorize user access.
 
 - [ProfileClient](https://github.com/Altinn/altinn-notifications/tree/main/src/Altinn.Notifications.Integrations/Profile/ProfileClient) 
   consumes Altinn Profile's internal API to retrieve contact points for individuals.
 - [RegisterClient](https://github.com/Altinn/altinn-notifications/blob/main/src/Altinn.Notifications.Integrations/Register/RegisterClient.cs)
-  consumes Altinn Register's internal API to retrieve the official contact points for organizations.
+  consumes Altinn Register's internal API to retrieve the official and user registered contact points associated with organizations.
+- [AuthorizationService](https://github.com/Altinn/altinn-notifications/blob/main/src/Altinn.Notifications.Integrations/Authorization/AuthorizationService.cs)
+  consumes Altinn Authorization's internal API to verify that all users with registered contact points for an organization are authorized. The decision request will ask
```

Comment:
Could we change to decision API? I guess it's only a decision endpoint if we're being clear, but to show we are using PDP. 
```suggestion
  consumes Altinn Authorization's Decision API to verify that all users with registered contact points for an organization are authorized. The decision request will ask
```